test/generators/component_generator_test.rb

```diff
@@ -57,6 +57,7 @@ def test_component_with_inline
     end
 
     assert_no_file "app/components/user_component.html.erb"
+    assert_no_file "component.html.erb"
```

Comment:
I added this test to check that the bug was fixed, but I'm not sure it makes sense to keep it in the codebase...